### DISCUSSION TITLE
mobile client not support latex

### DISCUSSION THREAD

**Discussion by ongiant** [community/community#47215] [Other Feature Feedback, Questions, & Ideas]
when I issued use latex format math equation, it's ok on my PC browser, but not ok on my mobile phone client.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FlyFlame ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
